provision-sm-md-lg/README.md

```diff
@@ -0,0 +1,71 @@
+# Provision VM Service Small Medium Large Example
+
+This example shows creating a provisioning dialog that modifies provisioning parameters
+
+## Architecture
+
+- `provision-vm-small-med-large.asl`: This workflow is run for every vm provisioned and is responsible for modifying the vm size. This will be used as a ManageIQ Service's Provisioning Entry Point.
+
+Set the ManageIQ provisioning service entry point to this script. This sample workflow is run for every provisioned Virtual Machine (VM), and modifies the VM size.
+
+## Add a Workflow Repository
+
+1. **Click Automation** > **Embedded Workflows** > **Repositories**
+2. **Click Configuration** > **Add new Repository**
+3. Provide the `Name` and `URL` for the repository. Example values: `Name: Example Workflows` and `URL: https://github.com/ManageIQ/workflows-examples`
+4. Click **Save**
+5. Verify that the workflow count column is not zero.
+
+## Create a Provisioning Dialog
+
+1. **Click Automation** > **Embedded Automate** > **Customization**
+2. **Click Service Dialogs** > **All Dialogs**
+3. **Click Configuration** > **Add a new Dialog**
+4. Provide a `Name` and `Description`. Example values: `Name = sm-med-lg dialog` and `Description = Dialog choosing vm size`
+5. Add a text box named `vm_name`.
+6. Add a Dropdown named `size`.
+7. Add dropdown values. Click the edit pencil icon for the "size" dropdown.
+8. **Click **Options** > **Entries** entering the following values:
+
+   |  Name | Value  |
+   |-------|--------|
+   |Large  | large  |
+   |Medium | medium |
+   |Small  | small  |
+
+9. Click **Save**
+
+## Create a Service Catalog Item
+
+1. Click **Service** > **Catalogs**
+2. **Catalogs** > **All Catalogs**
+3. **Configuration** > **Add a New Catalog**
+4. Provide a `Name` and `Description`. Example values: `Name = Provisioning` and `Description = Provisioning Services`
+5. Click **Save**
+6. Click **Catalog Items** > **All Catalog Items** > **Provisioning**
+7. Click **Configuration** > **Add a New Catalog Item**
+8. Choose a type. Example value: `Catalog Item Type = VMWare`
+9. Choose Tab **Basic Information**
+10. Enter a few values:
+11. `Name = Vm with Size`
+12. `Description = Provision a vm choosing size`
+13. `Display in Catalog = check`
+14. `Catalog = My Company/Provisioning`
+15. `Dialog = sm-med-lg dialog`
+16. `Provisioning Entry Point = Embedded Workflow, provision-vm-small-med-large.asl`
+17. Choose Tab **Request Info**
+18. Choose Tab **Catalog**
+19. A Naming > VM Name = new vm (NOTE: this will be ignored)
```

Comment:
Do not use will. This step is confusing. Ideally should not be using () and putting content in it.